### DISCUSSION TITLE
Handling conflicts from the client side an HTTP mode

### DISCUSSION THREAD

**Discussion by Rustam-23** [Mimetis/Dotmim.Sync#988] [Q&A]
hello!

I read the documentation and did not find a solution for the ability to resolve conflicts on the client side in HTTP mode. Maybe I missed something. Does a similar scenario like a Two sync trick exist?

**Comment by Mimetis** [comment]
Did you checked https://dotmimsync.readthedocs.io/Conflict.html#handling-conflicts-from-the-client-side ?

This method should work in HTTP mode as well

**Reply by Rustam-23** [reply]
Thanks for your reply. So the logic is the same? ConflictResolution is ServerWins and return from GetSyncConflictAsync() conflict.Result.LocalRow and conflict.Result.RemoteRow?

**Reply by Mimetis** [reply]
Yes
